content/docs/started/getting-started.md

```diff
@@ -11,46 +11,24 @@ This document provides information about setting up Kubeflow in various environm
 It's important that you have some knowledge of the following systems and tools:
 
 * [Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
-* [TensorFlow](https://www.tensorflow.org/get_started/)
-* [kustomize](https://kustomize.io/)
+* [Kustomize](https://kustomize.io/)
 
-### Kubeflow requirements
+## Installing Kubeflow
 
-Before installing Kubeflow, ensure that your environment meets the
-following requirements:
-
-* You need kustomize version {{% kustomize-min-version %}} or later. See the 
```

Comment:
Shouldn't we keep the info about the kustomize version?

Reply:
kustomize won't be needed necessarily (ie for the use cases where kubeflow is pre-installed and has the necessary components). But that isn't the reason I removed it - the same comment is in docs/components/misc/kustomize/, which is referenced in the "Installing command line tools" at the bottom of the page.